src/OSPSuite.Core/Journal/JournalPage.cs

```diff
@@ -48,7 +48,8 @@ public virtual Content Content
          set
          {
             _content = value;
-            ContentId = _content.Id;
+            if (_content != null)
```

Comment:
Resetting the content of the related item when loading a file. No need to keep that in memory!